### PR TITLE
OCPBUGS#11050: Explanation of  option for ImageSetConfiguration

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OCPBUGS-11050

Link to docs preview: https://57989--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-creating-image-set-config_installing-mirroring-disconnected

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/oc-mirror-creating-image-set-config.adoc [hunk 1]
```diff
@@ -61,7 +61,7 @@ mirror:
 <2> Set the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values, unless you are using the Technology Preview OCI feature.
 <3> Set the registry URL for the storage backend.
 <4> Set the channel to retrieve the {product-title} images from.
-<5> Add `graph: true` to generate the OpenShift Update Service (OSUS) graph image to allow for an improved cluster update experience when using the web console. For more information, see _About the OpenShift Update Service_.
+<5> Add `graph: true` to generate the `UpdateService` CR manifest that the OpenShift Update Service (OSUS) uses to provide a graph image for an improved cluster update experience when using the web console in a disconnected environment. For more information, see _About the OpenShift Update Service_.
```
Comment (shellyyang1989) [inline modules/oc-mirror-creating-image-set-config.adoc:64]: If setting `graph: true`, graph-data image that OSUS uses will be built and pushed to your mirror registry. In addition, the `UpdateService` CR manifest that the OpenShift Update Service (OSUS) uses will be generated as well. And it's not limited to web console only. It's easy for user to create OSUS instance using cli as well. 

cc @LalatenduMohanty @PratikMahajan for review. Thank you!

Comment (PratikMahajan) [reply modules/oc-mirror-creating-image-set-config.adoc:64]: agree with @shellyyang1989 ’s comment.

File: modules/oc-mirror-creating-image-set-config.adoc [hunk 2]
```diff
@@ -61,7 +61,7 @@ mirror:
 <2> Set the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values, unless you are using the Technology Preview OCI feature.
 <3> Set the registry URL for the storage backend.
 <4> Set the channel to retrieve the {product-title} images from.
-<5> Add `graph: true` to generate the OpenShift Update Service (OSUS) graph image to allow for an improved cluster update experience when using the web console. For more information, see _About the OpenShift Update Service_.
+<5> Add `graph: true` to build and push the graph image to the mirror registry. The `graph: true` option also generates the `UpdateService` CR manifest that the OpenShift Update Service (OSUS) uses to provide a graph image for an improved cluster update experience when using the web console and command-line interface (CLI) in a disconnected environment. For more information, see _About the OpenShift Update Service_.
```
Comment (shellyyang1989) [inline modules/oc-mirror-creating-image-set-config.adoc:64]: @PratikMahajan could you please suggest on the wording?

I think it's enough to mention:
Add `graph: true` to build and push the graph-data image to the mirror registry. The graph-data image is required to create OpenShift Update Service (OSUS)  OSUS application. The option also generates the `UpdateService` CR manifest that can be directly used by oc command-line interface (CLI) to create OpenShift Update Service (OSUS)  OSUS application.

Comment (PratikMahajan) [reply modules/oc-mirror-creating-image-set-config.adoc:64]: @shellyyang1989's suggestion accurately tells what oc-mirror is supposed to do. 
We should mention that the CR created by oc-mirror can be directly used by CLI. On the web-console front, the CR can be created using the form or we can directly apply the yaml. But form seems to be the default and easier to use on web console. we dont need to mention the form part in the docs. 
Lets go with @shellyyang1989's wording

**Comment by xenolinux** [comment]
@shellyyang1989 I updated the PR. I reworded a little to avoid the passive voice. Can you please check?

File: modules/oc-mirror-creating-image-set-config.adoc [hunk 3]
```diff
@@ -61,7 +61,7 @@ mirror:
 <2> Set the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values, unless you are using the Technology Preview OCI feature.
 <3> Set the registry URL for the storage backend.
 <4> Set the channel to retrieve the {product-title} images from.
-<5> Add `graph: true` to generate the OpenShift Update Service (OSUS) graph image to allow for an improved cluster update experience when using the web console. For more information, see _About the OpenShift Update Service_.
+<5> Add `graph: true` to build and push the graph image to the mirror registry. The graph-data image is required to create OpenShift Update Service (OSUS) applications. The `graph: true` field also generates the `UpdateService` CR manifest. The `oc` command-line interface (CLI) can use the `UpdateService` CR manifest to create OSUS applications. For more information, see _About the OpenShift Update Service_.
```
[the remark this answers is not in the document]
Comment (shellyyang1989) [reply modules/oc-mirror-creating-image-set-config.adoc:64]: Expect for ^ minor issue, it looks good to me. Thank you.

[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/oc-mirror-creating-image-set-config.adoc]: @shellyyang1989 Done.

**Review by snarayan-redhat — COMMENTED** [review]
Looks good.

**Comment by michaelryanpeter** [comment]
@xenolinux Great work!

**Pull request merged**